src/test/java/dog/pawbook/logic/commands/EnrolCommandTest.java

```diff
@@ -138,7 +138,7 @@ public void execute_enrolManyDogsManyPrograms_failure() {
         expectedModel.setEntity(ID_FIFTEEN, new Program(editedProgram1.getName(),
                 editedProgram1.getSessions(), editedProgram1.getTags(), updatedEnrolledDogs1));
 
-        HashSet<Integer> updatedEnrolledDogs2 = new HashSet<>(editedProgram1.getDogIdSet());
+        HashSet<Integer> updatedEnrolledDogs2 = new HashSet<>(editedProgram2.getDogIdSet());
```

Comment:
this one kinda violates the naming conventions, maybe drop the number for the one above and just reuse the same variable

Reply:
sorry yk what do you mean by this, it was bc yt accidentally put 1 instead of 2 (its by right supposed to be 2), but bc it was a failure test it still can pass the checks, so i j changed it for her!! but all the other testing functions use ...1 and ...2 if there are multiple programs involved!!

Reply:
I did this slipped my checks, but coding standards still need to be adhered to even for test code, it looks like bad practice to name variables with the number at the back, and it is fine to repurpose a variable after it is done. And I think if after these kind of changes, there are 2 highly similar blocks of code, maybe its time to extract a function out?

Reply:
okay, ill work on that in a separate commit!! i will add more failure tests first, thank u boss!! :D